requirements/edx/base.in

```diff
@@ -96,7 +96,6 @@ glob2                               # Enhanced glob module, used in openedx.core
 gunicorn==19.0
 help-tokens
 html5lib                            # HTML parser, used for capa problems
-ipaddr==2.1.11                      # Ip network support for Embargo feature
```

Comment:
Even though the `ipaddress` backport package is currently inherited via `requirements/edx-sandbox/shared.txt`, since we use it directly in the code I'd like to see it listed explicitly here in case that changes in the future.

Reply:
Ok, done :-)